### PR TITLE
adjusted readme file

### PR DESCRIPTION
**Pull request by RoyNyaga**

### CODE REVIEW

**Comment by Oghenebrume50** [comment]
A very good start to this project, awesome work just a few changes to be made.

The content in the div below the navbar with class="hero-image" should follow the guideline below

- [ ] There should be three links
- [ ] Each link is an <a> HTML tag, with a <div> tag inside it.
- [ ] The <div> tag  inside the <a> tag should use an image as background, the text on the original page is actually an image

- [ ] Use the main tag for the main part of the page
- [ ]Move the `<aside>` tag into the footer and follow the exact same guidelines above  to place the images very well

Correct those and submit for re-review

Also, I noticed you merged your last pull request to master before submitting, please this is not the way project submission work, you are to merge after review, not before a review

**Comment by RoyNyaga** [comment]
I added the main tag and used 3 links in the main in which i embedded one img tag each. I could not do the other changes because they are opposite to what is on the original site. My friends had this same problem and they had to report to the student support team before it was approved. If placing the aside tag inside the footer and using images as background inside the aside tag are requirements for this project, then i think the project requirements where not well established and needs to be changed. Please do well to inspect the page for yourself. Thanks for understanding

**Comment by mosaaleb** [comment]
Reviewing

File: index.html [hunk 1]
```diff
@@ -50,26 +50,27 @@
       </form>
```
Comment (mosaaleb) [inline index.html:50]: For simple UI components like search field, there is no need to include the `<form>` tag. See [this](https://stackoverflow.com/questions/3294572/is-input-well-formed-without-a-form)

**Review by mosaaleb — APPROVED** [review]
### Congratulations @RoyNyaga :clap:
#### You can merge into master :+1: 

Please consider my comment before merging, you don't have to submit a code review for this.

**Also** consider reading about when to use the html `` tag vs css `background-img` property when working with images. See [this](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) and please apply this knowledge to the upcoming projects.

Good luck with the next projects.

### Happy Coding :computer:

File: index.html (hunk 1 above)
Comment (RoyNyaga) [reply index.html:50]: Thanks sir, the suggestions are very helpful.

**Pull request merged**